src/cordova/test.action.mjs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 The Outline Authors
```

Comment:
Should this be under apple instead? We are only testing the apple code anyway.
/cc @daniellacosse

Reply:
Ok, I guess it's fine to keep it here if android will be here too. I was thinking we were going to have separate actions